python/test/unit/language/test_core.py

```diff
@@ -4962,10 +4962,8 @@ def test_fp8_dot_acc(in_type_str, low_precision_acc, device):
     ref_out = torch.matmul(th_a, th_b).to(torch.float32)
     if in_type_str == 'float8e4nv':
         torch.testing.assert_close(ref_out, C, rtol=0.01, atol=0.01)
-    elif low_precision_acc > 32:
-        torch.testing.assert_close(ref_out, C, rtol=1e-3, atol=1e-3)
     else:
-        torch.testing.assert_close(ref_out, C)
+        torch.testing.assert_close(ref_out, C, rtol=1e-3, atol=1e-3)
```

Comment:
enlarge the equal tolerance for low_precision_acc <= 32 since there will be a slight calculation accuracy error in fp8 mma

Reply:
The point of this test is to make sure when low_precision_acc is set to 0 we have the right precision

Reply:
But I found that there is still a precision problem even if low_precision_acc is 0

Reply:
that means something is wrong since this test was passing?

Reply:
I wonder if mmav2 f8 has precision problems. In anyway I think on hopper addressing this will solve it:
https://github.com/openai/triton/pull/3479#discussion_r1544113771

Reply:
oh, I mean if not promote, fp8 mma on sm89 will have a slight precision decrease even if low_precision_acc is 0. It makes me quite confused...